TemporaryParser/TPBISettingPreviewer.class.st

```diff
@@ -0,0 +1,262 @@
+Class {
+	#name : #TPBISettingPreviewer,

```

Comment:
Maybe we should move this class to another package. Something like
TemporaryParser-Extras?